build/vsts_build.yaml

```diff
@@ -77,6 +77,9 @@ phases:
     FullVstsBuildNumber: $[dependencies.Initialize_Build.outputs['updatebuildnumber.FullVstsBuildNumber']]
     VsTargetChannel: $[dependencies.Initialize_Build.outputs['updatebuildnumber.VsTargetChannel']]
     VsTargetMajorVersion: $[dependencies.Initialize_Build.outputs['updatebuildnumber.VsTargetMajorVersion']]
+    LocalizedLanguageCount: "13"
+    DesktopTargetFramework: "net472"
```

Comment:
Where does this variable is used?

Reply:
That variable is unused. I added it today, but then stopped using it, and forgot to remove it.

Reply:
working on a new cleanup PR with this (and other changes that didn't make 5.5)...to merge into dev. This is fixed in next PR.